### ISSUE TITLE
Confusing debug log message

### ISSUE THREAD

**Issue by bruno-garcia** [getsentry/sentry-dotnet#3200]
https://github.com/search?q=repo%3Agetsentry%2Fsentry-dotnet+%22Failed+to+report+an+error+on+a+session+because+there+is+none+active.%22&type=code

Please reword this as it gets people confused that something went wrong. 
Nothing went wrong so no reason to sound alarm.

Suggestions: "Session tracking turned off so no session update will be reported. Consider using AutoSessionTracking to get crash free rate" or something.

**Comment by bitsandfoxes** [comment]
I encountered the same thing coincidentally in Unity too https://github.com/getsentry/sentry-unity/pull/1576
